team-2018.html

```diff
@@ -212,14 +212,14 @@ <h3>Dheeraj Gunda</h3>
                     </div>
                 </div>
                 <hr class="featurette-divider">
-                <!-- <div class="jumbotron text-info" style="background: none">
+                <div class="jumbotron text-info" style="background: none">
```

Comment:
Not just this. Add a button for 2017 team **and** one for 2018 in `team.html` & `team-2018.html`, both.